config/cdot_ppmBsm.properties

```diff
@@ -22,10 +22,10 @@ privacy.filter.geofence.ne.lon=-102.051701
 privacy.topic.consumer=topic.OdeBsmJson
 privacy.topic.producer=topic.FilteredOdeBsmJson
 
-group.id=0
+group.id=CDOT_PPM_BSM
 
 # For testing purposes, use one partition.
-privacy.kafka.partition=0
+# privacy.kafka.partition=0
```

Comment:
Let's add the same property to allow larger data consumption per poll that we added to the acm module.

Reply:
That's the max.partition.fetch.bytes property, right? Should we just add that to our properties files or the ppmBsm & ppmTim properties files as well?

Reply:
This property has been added to the following files:
- cdot_ppmBsm.properties
- cdot_ppmTim.properties
- ppmBsm.properties
- ppmTim.properties